driver_author_analysis.py

```diff
@@ -5,9 +5,10 @@
 from lib.analysis.thread.hypergraph import generate_hyperedge_distribution
 from lib.input.mbox.keyword_clustering import generate_kmeans_clustering
 from lib.input.mbox.keyword_digest import generate_keyword_digest
+from lib.analysis.author.community import vertex_clustering
 
 mailbox_list = ['lkml', 'opensuse-kernel', 'opensuse-features', 'opensuse', 'opensuse-bugs', 'opensuse-factory', 'sakai-devel']
-mailbox_list = ['sakai-devel', 'lkml', 'opensuse', 'opensuse-bugs']
```

Comment:
it might be better to leave all the mailing lists here, at least as a comment line.